content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -151,7 +153,14 @@ stages:
 by the user with the `--name` (`-n`) option of `dvc run`. Each stage can contain
 the possible following fields:
 
-- `cmd` (always present): Executable command defined in this stage
+- `cmd` (always present): Executable commands defined in this stage. Each
+  command is passed as is to the shell. The command can be either a single
+  command or a list of commands. In that latter case:
+  - the commands are run one after the other
+  - the failure of one given command will stop the run of the stage and raises
+    an error. Thus that all the commands before failure are well executed.
+    > in the `dvc.yaml` example above, the `python train.py` command will be
+    > executed only if the command before succeeds, _i.e_ if git is clear
```

Comment:
- [ ] But the details of how commands are executed should go somewhere in the `dvc repro` command reference instead of here. This is just a description of the field in dvc.yaml 🙂

Reply:
Unresolving this too. I don't see any new changes above or in the repro ref.

Reply:
sorry i am used to resolve as soon I have taken them into account locally because the "outdated" from github should eventually show if something actually changed

Reply:
No problem, I guess we were both online at the same time. It happens.